tsup.config.ts

```diff
@@ -2,13 +2,11 @@ import { defineConfig } from 'tsup';
 
 export default defineConfig([
   {
+    format: ['cjs', 'esm'],
     entry: ['src/index.ts'],
-    treeshake: true,
```

Comment:
I wasn't part of the conversation from the beginning, but on the surface I don't think removing this `treeshake` property is correct.

Reply:
I can put it back for sure.

Reply:
Done